questions.md

```diff
@@ -0,0 +1,4 @@
+# Question on Pull Request
+1. Please what is the difference between git merge &  pull request.
```

Comment:
Git merge incorporates a branch into the branch you are currently on: vhttps://git-scm.com/docs/git-merge
A pull request is a collaboration feature, where you submit a branch (or several!) for review from your team.